datasketches/src/theta/mod.rs

```diff
@@ -32,4 +32,6 @@
 mod hash_table;
 mod sketch;
 
+pub use self::hash_table::ResizeFactor;
 pub use self::sketch::ThetaSketch;
+pub use self::sketch::ThetaSketchBuilder;
```

Comment:
To @ZENOTME,

`ThetaSketchBuilder` and `ResizeFactor` are needed because `ThetaSketch::builder` exposes the builder, which has a `resize_factor` method that only accepts a `ResizeFactor` instance.